source/_integrations/litterrobot.markdown

```diff
@@ -0,0 +1,111 @@
+---
+title: Litter-Robot
+description: Instructions on how to integrate a Litter-Robot WiFi-enabled, automatic, self-cleaning litter box to Home Assistant.
```

Comment:
We use the `Wi-Fi` spelling.

<https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>